packages/docs/src/components/docsearch/doc-search.tsx

```diff
@@ -71,8 +73,16 @@ export const DocSearch = component$((props: DocSearchProps) => {
     status: 'idle',
     snippetLength: 10,
   });
-
   const searchButtonRef = useSignal<Element>();
+  const isOpen = useSignal(false);
+
+  useTask$((ctx) => {
+    ctx.track(() => {
+      if (state.isOpen != isOpen.value) {
+        isOpen.value = state.isOpen;
```

Comment:
just put the isOpen signal in the store and change everything to use .value

Reply:
nice did not know i could do this!

Reply:
the current component currently live in place were the CSS will  hide  when the viewport is small! will see if there a  workaround this but I do see why not having duplicate since it will lead to weird behaviour if you were to change the viewport

Reply:
complted